container/configs/named.conf.options.authoritative

```diff
@@ -0,0 +1,30 @@
+// Configure the IPs to listen on here.
```

Comment:
I like the proposal of having this in a separate options that can be loaded if the user wants to turn it into an authoritative (vs a recursive).

Definitely going to implement this.